sling/nlp/wiki/wikipedia-links.cc

```diff
@@ -157,6 +158,46 @@ class WikipediaLinkExtractor : public task::DocumentProcessor {
 
 REGISTER_TASK_PROCESSOR("wikipedia-link-extractor", WikipediaLinkExtractor);
 
+// Collect fact targets from items and output aggregate target counts.
+class FactTargetExtractor : public task::FrameProcessor {
+ public:
+  void Startup(task::Task *task) override {
+    accumulator_.Init(output());
+  }
+
+  void Process(Slice key, const Frame &frame) override {
+    // Accumulate fact targets for the item.
+    Store *store = frame.store();
+    for (const Slot &slot : frame) {
+      if (slot.name == Handle::isa()) continue;
+      if (slot.name == n_lang_) continue;
+
+      Handle target = store->Resolve(slot.value);
+      if (!store->IsFrame(target)) continue;
+
+      Text id = store->FrameId(target);
+      if (id.empty()) continue;
+
+      if (!store->IsFrame(target)) continue;
```

Comment:
Looks like this line might be redundant.

Reply:
Oops. Removed.